### DISCUSSION TITLE
[Feature] Improved People Merge - Zoom / See Original

### DISCUSSION THREAD

**Discussion by BloodBlight** [immich-app/immich#14854] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When merging people, the face preview isn't always clear enough to really tell. 

Could we add a zoom icon to see the whole image?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by ireun** [comment]
Yes please! Some idead:
* let us enlarge a photo of a person being merged (a 'magnifying glass' icon maybe?)
* let open person-face-page (an 'info' icon maybe?)

**Comment by bo0tzz** [comment]
#17056
